index.js

```diff
@@ -2,31 +2,33 @@
 const got = require('got');
 const registryUrl = require('registry-url');
 
-function get(keyword, size) {
+function get(keyword, options) {
 	if (typeof keyword !== 'string') {
 		return Promise.reject(new TypeError('Keyword must be a string'));
 	}
 
 	keyword = encodeURIComponent(keyword);
 
-	const url = `${registryUrl()}-/v1/search?text=keywords:${keyword}&size=${size}`;
+	const url = `${registryUrl()}-/v1/search?text=keywords:${keyword}&size=${options.size}`;
 
 	return got(url, {json: true}).then(response => response.body);
 }
 
-module.exports = keyword => {
-	return get(keyword, 250).then(data => {
+module.exports = (keyword, options) => {
+	options = options || {size: 250};
```

Comment:
Use `Object.asssign`

Reply:
We should also validate that the number is between `1` and `250`, and throw a user-friendly error if not.

Reply:
Done.